# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/trunk/webrtc/voice_engine/channel_proxy.cc
# Commit: 5d2899adc305
# Full Hash: 5d2899adc3057fe4df9dd454887c1832f54d91d9
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2018-11-09 10:01:40
# Regressor Bug: 1376873
# File Overlap Count: 4
# Description:
#   Bug 1376873 - Use Call interface in AudioConduit; r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D7441
# ==============================================================================

diff -r df960ea2d29e -r 5d2899adc305 media/webrtc/trunk/webrtc/voice_engine/channel_proxy.cc
--- a/media/webrtc/trunk/webrtc/voice_engine/channel_proxy.cc	Mon Mar 26 16:19:23 2018 -0400
+++ b/media/webrtc/trunk/webrtc/voice_engine/channel_proxy.cc	Tue Aug 21 13:39:53 2018 -0400
@@ -87,7 +87,7 @@
 }
 
 void ChannelProxy::SetReceiveCsrcAudioLevelIndicationStatus(bool enable, int id) {
-  //RTC_DCHECK(worker_thread_checker_.CalledOnValidThread());
+  RTC_DCHECK(worker_thread_checker_.CalledOnValidThread());
   int error = channel()->SetReceiveCsrcAudioLevelIndicationStatus(enable, id);
   RTC_DCHECK_EQ(0, error);
 }
